packages/api-reference/src/components/Content/Introduction/Introduction.vue

```diff
@@ -102,4 +120,11 @@ onMounted(() => config.value.onLoaded?.())
   position: sticky;
   top: calc(var(--refs-header-height) + 24px);
 }
+
+.links {
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+  margin-bottom: 18px;
+}
```

Comment:
let’s put the div in a seperate component (maybe `<LinkList />`), having the div markdown here looks a bit strange between all the nice Vue components

Reply:
i'll let @cameronrohani jump here according to the given style within tag